samples/react-native/ios/sentryreactnativesample.xcodeproj/project.pbxproj

```diff
@@ -282,10 +282,14 @@
 			inputFileListPaths = (
 				"${PODS_ROOT}/Target Support Files/Pods-sentryreactnativesample/Pods-sentryreactnativesample-frameworks-${CONFIGURATION}-input-files.xcfilelist",
 			);
+			inputPaths = (
```

Comment:
q: Are the `inputPaths` and `outputPaths` additions needed?

Reply:
Nope but they get added automatically by the latest Xcode 😅